tests/conftest.py

```diff
@@ -76,6 +76,15 @@ def songs_csv():
         return song_csv_file.read().encode("utf-8")
 
 
+@fixture(scope="session")
+def songs_csv_custom_separator():
+    """
+    Runs once per session. Provides the content of songs_custom_delimiter.csv from read..
```

Comment:
Is this completed? `from read` what? 🦖 
```suggestion
    Runs once per session. Provides the content of songs_custom_delimiter.csv from read..
```